lib/appsignal/plug.ex

```diff
@@ -21,13 +21,16 @@ if Appsignal.plug?() do
               conn
               |> Plug.Conn.put_private(:appsignal_transaction, transaction)
               |> Plug.Conn.register_before_send(fn conn ->
-                Appsignal.Plug.finish_with_conn(transaction, conn)
+                Appsignal.Plug.try_set_action(transaction, conn)
```

Comment:
If this wasn't called when it receives an unhandled exit, that means it will still not call this even if we now finish the transaction correctly, right?
So we'd get errors reported without an action name if even if they originated from this function.

Do we know the action name before the request has completed? Can we set it earlier in the request instead?